src/rgw/rgw_op.cc

```diff
@@ -5339,9 +5339,12 @@ void RGWPutACLs::execute()
     *_dout << dendl;
   }
 
-  op_ret = policy->rebuild(store->ctl()->user, &owner, new_policy);
-  if (op_ret < 0)
+  std::string err_msg;
```

Comment:
you can pass ``s->err.message`` directly into the function. no need for this temp variable.

Reply:
@yuvalif 
done~